src/components/AuthForm/FormButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { ButtonHTMLAttributes } from "react";
+
+interface FormButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  loading?: boolean;
+}
+
+const FormButton = ({
+  loading = false,
+  children,
+  ...props
+}: FormButtonProps) => {
+  return (
+    <button
+      disabled={loading || props.disabled}
```

Comment:
구조분해할당을 disabled도 해주시면 좋을 것 같아요.
```suggestion
      disabled={loading || disabled}
```